README.md

```diff
@@ -639,6 +639,21 @@ function Header() {
 }
 ```
 
+It is generally a good idea to use the above method for importing assets, because:
+
+- throws a compilation error when an asset is missing.
+- filenames include hashing for browser caching.
```

Comment:
Which filenames? I'm not sure this is clear to someone who doesn't already know what it's talking about.